### PR TITLE
Change Taxii V2 and Crowdstrike Falcon Intel Feed to work with lastRun

### PR DESCRIPTION
**Pull request by gal-berger**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35698

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 1]
```diff
@@ -4646,6 +4646,98 @@ def test_invalid_b_type_init(self):
...
+        mocker.patch('CommonServerPython.get_demisto_version', return_value={"version": "6.1.0"})
+        mocker.patch.object(demisto, 'getIntegrationContext', return_value={1: "first indicator"})
+        result = get_feed_last_run()
+        assert result == {1: "first indicator"}
+
+    def test_get_last_run_in_6_2_when_get_last_run_has_no_results(self, mocker):
+        """
+        Given: 6.2.0 environment
+        When: Fetch indicators when getLastRun and getIntegrationContext are empty
+        Then: {}
```
Comment (yuvalbenshalom) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:4682]:
```suggestion
        Then: function will return empty dict
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 2]
```diff
@@ -4646,6 +4646,98 @@ def test_invalid_b_type_init(self):
...
+        mocker.patch.object(demisto, 'getIntegrationContext', return_value={})
+        mocker.patch.object(demisto, 'getLastRun', return_value={})
+        result = get_feed_last_run()
+        assert result == {}
+
+    def test_get_last_run_in_6_2_when_get_last_is_empty_and_get_integration_is_not(self, mocker):
+        """
+        Given: 6.2.0 environment
+        When: Fetch indicators when getLastRun is empty and getIntegrationContext has results.
+        Then: {}
```
Comment (yuvalbenshalom) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:4696]:
```suggestion
        Then: function will return empty dict
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 3]
```diff
@@ -4646,6 +4646,98 @@ def test_invalid_b_type_init(self):
...
+        mocker.patch('CommonServerPython.get_demisto_version', return_value={"version": "6.2.0"})
+        mocker.patch.object(demisto, 'getIntegrationContext', return_value={})
+        mocker.patch.object(demisto, 'getLastRun', return_value={})
+        result = get_feed_last_run()
+        assert result == {}
+
+    def test_get_last_run_in_6_2_when_get_last_is_empty_and_get_integration_is_not(self, mocker):
+        """
+        Given: 6.2.0 environment
+        When: Fetch indicators when getLastRun is empty and getIntegrationContext has results.
```
Comment (yuvalbenshalom) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:4695]: "setLastRun is empty and getIntegrationContext has results." is part of the "Given"

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 4]
```diff
@@ -4646,6 +4646,98 @@ def test_invalid_b_type_init(self):
...
+        from CommonServerPython import get_feed_last_run
+        mocker.patch('CommonServerPython.get_demisto_version', return_value={"version": "6.1.0"})
+        mocker.patch.object(demisto, 'getIntegrationContext', return_value={1: "first indicator"})
+        result = get_feed_last_run()
+        assert result == {1: "first indicator"}
+
+    def test_get_last_run_in_6_2_when_get_last_run_has_no_results(self, mocker):
+        """
+        Given: 6.2.0 environment
+        When: Fetch indicators when getLastRun and getIntegrationContext are empty
```
Comment (yuvalbenshalom) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:4681]: "getLastRun and getIntegrationContext are empty" is part of the given

File: Packs/FeedTAXII/ReleaseNotes/1_0_9.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Deprecated **reset_fetch_command** command.
+- Updated the Docker image to: *demisto/taxii2:1.0.0.19258*.
+- Maintenance and stability enhancements.
```
Comment (yuvalbenshalom) [inline Packs/FeedTAXII/ReleaseNotes/1_0_9.md:6]:
```suggestion
```

File: Packs/FeedTAXII/ReleaseNotes/1_0_9.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Deprecated **reset_fetch_command** command.
```
Comment (yuvalbenshalom) [inline Packs/FeedTAXII/ReleaseNotes/1_0_9.md:4]: you should add what should be used instead, from Cortex XSOAR version 6.2

File: Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/2_0_1.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### CrowdStrike Falcon Intel Feed Actors
+- Maintenance and stability enhancements.
```
Comment (yuvalbenshalom) [inline Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/2_0_1.md:3]: replace this with added support for reset last run for Cortex XSOAR version 6.2

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py (hunk 1 above)
Comment (gal-berger) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py]: done

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py (hunk 2 above)
Comment (gal-berger) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py]: done

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py (hunk 3 above)
Comment (gal-berger) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:4695]: done

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py (hunk 4 above)
Comment (gal-berger) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:4681]: done

File: Packs/FeedTAXII/ReleaseNotes/1_0_9.md (hunk 6 above)
Comment (gal-berger) [reply Packs/FeedTAXII/ReleaseNotes/1_0_9.md:4]: done

File: Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/2_0_1.md (hunk 7 above)
Comment (gal-berger) [reply Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/2_0_1.md]: done

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/FeedTAXII/ReleaseNotes/1_0_9.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Deprecated **reset_fetch_command** command. From Cortex XSOAR version 6.2: Use **Re-fetch indicators** button in the feed's instance settings instead.
```
Comment (bakatzir) [inline Packs/FeedTAXII/ReleaseNotes/1_0_9.md:4]:
```suggestion
- Deprecated the ***reset_fetch_command*** command. From Cortex XSOAR version 6.2 use the **Re-fetch indicators** button in the feed instance settings instead.
```

**Review by bakatzir — APPROVED** [review]

File: Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/2_0_1.md [hunk 9]
```diff
@@ -2,4 +2,4 @@
 ##### CrowdStrike Falcon Intel Feed Actors
 - Upgraded the Docker image to: *demisto/python3:3.9.5.20958*.
 ##### CrowdStrike Indicator Feed
-- Upgraded the Docker image to: *demisto/python3:3.9.5.20958*.
+- Upgraded the Docker image to: *demisto/python3:3.9.5.20958*.
```
Comment (bakatzir) [inline Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/2_0_1.md:5]: please reverted the changes in the old rn file

File: Packs/FeedTAXII/ReleaseNotes/1_0_9.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
...
+- Updated the Docker image to: *demisto/taxii2:1.0.0.19258*.
```
Comment (bakatzir) [inline Packs/FeedTAXII/ReleaseNotes/1_0_9.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/taxii2:1.0.0.19258*.
```

**Review by bakatzir — APPROVED** [review]

**Pull request merged**